doc/specs/#1502 - Advanced Tab Switcher/spec.md

```diff
@@ -120,21 +125,27 @@ There are two _dismissal_ keybinds:
 1. <kbd>enter</kbd> : brings the currently selected tab into focus and dismisses the UI.
 2. <kbd>esc</kbd> : dismisses the UI without changing tab focus.
 
-The following are ways a user can dismiss the UI, _whether or not_ the `Anchor` arg is provided to `OpenTabSwitcher`.
+The following are ways a user can dismiss the UI, _whether or not_ the `Anchor` key is provided to `openTabSwitcher`.
 
 1. The user can press a number associated with a tab to instantly switch to the tab and dismiss the switcher.
 2. The user can click on a tab to instantly switch to the tab and dismiss the switcher.
 3. The user can click outside of the UI to dismiss the switcher without bringing the selected tab into focus.
 4. The user can press any of the dismissal keybinds.
 
-If the `Anchor` arg is provided, then in addition to the above methods, the UI will dismiss upon the release of the `Anchor` key.
+If the `anchor` key is provided, then in addition to the above methods, the UI will dismiss upon the release of the `anchor` key.
 
-Pressing the `OpenTabSwitcher` keychord again will _not_ close the Switcher.
+Pressing the `openTabSwitcher` keychord again will _not_ close the Switcher.
 
 ### Most Recently Used Order
 
 We'll provide a setting that will allow the list of tabs to be presented in either _in-order_ (how the tabs are ordered on the tab bar), or _Most Recently Used Order_ (MRU). MRU means that the tab that the terminal most recently visited will be on the top of the list, and the tab that the terminal has not visited for the longest time will be on the bottom.
 
+There will be an argument for the `openTabSwitcher` action called `displayOrder`. This can be either `inOrder` or `mruOrder`. Making the setting an argument passed into `openTabSwitcher` would allow the user to have one keybinding to open an MRU Tab Switcher, and different one for the In-Order Tab Switcher. For example:
+```
+  {"keys": ["ctrl+tab"], "command": {"action": "openTabSwitcher", "anchor":"ctrl", "displayOrder":"mruOrder"}}
+  {"keys": ["ctrl+shift+p"], "command": {"action": "openTabSwitcher", "anchor":"ctrl", "displayOrder":"inOrder"}}

```

Comment:
What should happen if the user doesn't provide a `displayOrder` value? Should the default be in-order or MRU order?

Reply:
I'm sort of inclined to go with MRU as default, but that's because I'm used to seeing Tab Switchers in other apps have MRU default as well. I'm open to any other thoughts though!

Reply:
Yea I'm cool with that